internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -0,0 +1,2192 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#include "ChunkedSegmentSealedImpl.h"
```

Comment:
> is there a reason we create a new Class? This may bring difficulty for understanding. SealedSegment should be chunked by default, for compatibility reasons it can be only one chunk.
> 
> Using name SealedSegmentV2 could make more sense

SealedSegment keep a data pointer as a whole chunk so it's not chunked. We will change a lot of codes to make chunks work. So I add a new class that have many codes duplicated with SealedSegment. This class's functionality won't affect SealedSegment so we can rollback to SealedSegment if the chunked version does not work correctly.